src/Backend/GameLogic/GameManager.ts

```diff
@@ -365,7 +365,7 @@ class GameManager extends EventEmitter {
 
   public gameover$: Monomitter<boolean>;
 
-  private winners: string[];
+  private winners: EthAddress[] | undefined;
```

Comment:
When is this undefined? Won't it just be an empty array if no winners?

Reply:
and if the only time we get winners is from `getWinners()`, this will never be undefined? Also, Boolean([]) in JS is true, so need to be careful about evaluating it directly.

https://stackoverflow.com/questions/19146176/why-do-empty-javascript-arrays-evaluate-to-true-in-conditional-structures

Reply:
i agree this should be EthAddress[], ill commit that change

Reply:
i agree this should be changed, committing now